.vscode/wp-parsely.code-workspace

```diff
@@ -0,0 +1,152 @@
+{
+	"folders": [
+		{
+			"name": "wp-parsely",
+			"path": ".."
+		}
+	],
+	"launch": {
+		// Use IntelliSense to learn about possible attributes.
+		// Hover to view descriptions of existing attributes.
+		// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+		"version": "0.2.0",
+		"configurations": [
+			{
+				"name": "Listen for Xdebug",
+				"type": "php",
+				"request": "launch",
+				"port": 9003
+			},
+		]
+	},
+	"extensions": {
+		"recommendations": [
+			"dbaeumer.vscode-eslint",
+			"usernamehw.errorlens",
```

Comment:
I'm using Error Lens as well, bu I understand it can be noisy/overwhelming for some folks. But since this is just suggested, I'm fine with leaving it.

Reply:
Sounds Good. I also think we should keep/suggest it and devs can then take their own decision.